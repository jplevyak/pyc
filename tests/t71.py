print chr(65)
print ord('A')
print hex(0x5A7D)
