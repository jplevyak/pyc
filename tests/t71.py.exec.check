A
65
0x5a7d
