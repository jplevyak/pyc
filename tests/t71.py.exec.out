A
65
0x5A7D
